src/front-end/src/components/MentorForm.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
-import { Form, Button, Input, Tooltip, Select } from 'antd';
+import { Form, Button, Input, Select } from 'antd';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { InfoCircleOutlined } from '@ant-design/icons';
```

Comment:
Thanks for cleaning the codebase! Since we're not importing something extraneous anymore, the line above can be deleted too. I'll make a commit to delete it.